lib/script_base.rb

```diff
@@ -56,7 +56,7 @@ def run
     stderr.puts "*Task*: `#{result.subtask}`"
     stderr.puts "*UUIDs*: #{result.uuids.map { |uuid| "`#{uuid}`" }.join(', ')}"
     if result.messages.present?
-      stderr.puts "*Messages*:\n#{result.messages.map { |message| "* #{message}" }.join("\n")}"
+      stderr.puts "*Messages*:\n#{result.messages.map { |message| "    • #{message}" }.join("\n")}"
```

Comment:
confirmed, looks like slack wont auto-parse bullets in the API

https://stackoverflow.com/questions/68482844/how-do-i-post-a-bulleted-list-using-the-slack-api